connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -262,44 +234,15 @@ public void testRestartTask() throws Exception {
     }
 
     @Test
-    public void testRestartTaskFailureOnStop() throws Exception {
```

Comment:
Yes this test was simulating handling of exceptions from `Worker.stopAmdAwaitTask()` throwing which shouldn't happen anymore. Previously it would if the `taskId` was not owned by the worker.